4001-5000/LIT4310Prayer.xml

```diff
@@ -60,6 +36,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract>
+            <p>Collection of prayers, which were used as <term key="Magic">magic prayers</term>, each of which beginning with 
+               <foreign xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡</foreign> and often continued with 
+               <foreign xml:lang="gez">አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት፡</foreign> or
+               <foreign xml:lang="gez">አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ስምከ፡</foreign> and with <foreign xml:lang="gez">ʾasmāt</foreign>, 
+               some of which are styled according to the Syriac alphabet.</p>
```

Comment:
I believe this is the "Hebrew" alphabet as attested in Ps 118. It seems to be a mix of primarily Hebrew letter names with some that are closer to Syriac and some that are simply different :)